status-im/status-react
# Delivered messages marked in private chats

fixes #11439

### Summary

Client listens for `message.delivered` signal from `status-go` and mark messages with double-check icon:

![Screenshot 2021-02-16 at 14 25 48](https://user-images.githubusercontent.com/5786310/108062793-f38ad500-7062-11eb-8d00-bd77ca339bc7.png)

Works only for 1-1 chats

#### Platforms
- Android
- iOS

#### Areas that maybe impacted

##### Functional

- 1-1 chats
- public chats
- group chats

### Steps to test
- Open private chat between Alice and Bob
- Open client for Alice and send message to Bob
- make sure message marked as sent but not delivered
- Login into Bob client
- make sure message now marked as delivered for Alice

status: ready
